plugins/woo-ai/includes/class-woo-ai-admin-notices.php

```diff
@@ -18,4 +18,11 @@ class Woo_AI_Admin_Notices {
 	public function woocoommerce_not_installed() {
 		include_once dirname( __FILE__ ) . '/views/html-admin-missing-woocommerce.php';
```

Comment:
I know this isn't from this PR, but  I think we want to update the associated view with this other notice. It refers to the Beta Tester plugin

Reply:
Updated in https://github.com/woocommerce/woocommerce/pull/38452/commits/284c95c64ac2c99b12a64a994ba4c810997c867f